src/main/java/com/misterveiga/cds/listeners/MessageListener.java

```diff
@@ -202,6 +209,13 @@ public void onMessageReceived(final MessageReceivedEvent event) {
 	private void scanMessage(final Message message, final int i) {
 
 		final String messageText = message.getContentRaw();
+		
+		if (message.getChannel().getIdLong() == Properties.CHANNEL_BAN_REQUESTS_QUEUE_ID) {
+			if (!RoleUtils.isAnyRole(message.getMember(), RoleUtils.ROLE_SERVER_MANAGER, RoleUtils.ROLE_SENIOR_MODERATOR,
```

Comment:
Why are we blocking staff from approving ban requests with this condition? Shouldn't it be RoleUtils.isAny instead of !RoleUtils.isAny?

Reply:
This is blocking staff from being affected by the restricted format